.github/workflows/build.yml

```diff
@@ -0,0 +1,110 @@
+name: build binaries (test)
```

Comment:
```suggestion
name: build binaries
```
guessing

Reply:
oooh wait i see this is a separate job for testing that the build works on a workflow dispatch. nvm

Reply:
Yes I left it there for debugging.